FriesBSCam.csproj

```diff
@@ -44,10 +48,15 @@
       <Private>False</Private>
       <SpecificVersion>False</SpecificVersion>
     </Reference>
+    <Reference Include="websocket-sharp, Version=1.0.2.59611, Culture=neutral, PublicKeyToken=5660b08a1845a91e, processorArchitecture=MSIL">
+      <SpecificVersion>False</SpecificVersion>
+      <HintPath>..\..\..\..\..\Program Files (x86)\Steam\steamapps\common\LIV\resources\VIVR\VIVR_Data\Managed\websocket-sharp.dll</HintPath>
```

Comment:
This doesn't look like it will be portable

Reply:
yeah worth looking into!